Library/Formula/icecream.rb

```diff
@@ -0,0 +1,92 @@
+class Icecream < Formula
+  homepage ""
+  url "https://github.com/icecc/icecream/archive/1.1rc1.tar.gz"
+  version "1.1rc1"
+  sha1 "ccde08f67297122710270b440492a528df042f52"
+
+  depends_on "autoconf" => :build
+  depends_on "automake" => :build
+  depends_on "libtool" => :build
+  depends_on "docbook2X" => :build
+
+  def install
+    system "./autogen.sh"
+
+    system "./configure", "--disable-debug",
+                          "--disable-dependency-tracking",
+                          "--disable-silent-rules",
+                          "--prefix=#{prefix}"
+
+    system "make", "install"
+    (prefix+'org.opensuse.icecc.plist').write iceccd_plist
+    (prefix+'org.opensuse.icecc-scheduler.plist').write scheduler_plist
+  end
+
+  def caveats
+    return <<-EOS
```

Comment:
it seems I missed that return. I didn't realize undent was available here. It'll look much better that way.